### DISCUSSION TITLE
More social icons & cuts

### DISCUSSION THREAD

**Discussion by orzklv** [community/community#47054] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I've mentioned that recently, the twitter link has been replaced with 5 custom social network links. Also, if we add links to twitter or instagram pages, it automatically cuts the baseUrl by leaving only username in there and places an icon before username. However, this feature works only if you use twitter or instagram as far as i know. However, in central or pacific asia, there are many mainstream social messengers like telegram, line. Please, can you add more icons and social messenger link detections to profile social links? That would be really great to see as an active telegram user! ☺️🙏

**Comment by pjpscriv** [comment]
@orzklv As a fill-in solution, I've created a chrome extension that does this: https://chromewebstore.google.com/detail/github-social-icons/ledidlccnpfooopagmkgihgibfbnbbnp - it currently does Telegram + a few others, and open to requests for more :)
